charts/descheduler/values.yaml

```diff
@@ -60,6 +60,9 @@ leaderElection: {}
 cmdOptions:
   v: 3
 
+# Recommended to use the latest Policy API version supported by the Descheduler app version
+deschedulerPolicyAPIVersion: "descheduler/v1alpha2"
```

Comment:
based on @ingvagabund suggestion, let's stick to v1alpha1 for now until we have done thorough testing
```suggestion
deschedulerPolicyAPIVersion: "descheduler/v1alpha1"
```

Reply:
updated as suggested